config/config.go

```diff
@@ -61,9 +61,9 @@ const LedgerFilenamePrefix = "ledger"
 // It is used to recover from node crashes.
 const CrashFilename = "crash.sqlite"
 
-// CompactCertFilename is the name of the compact certificate database file.
-// It is used to track in-progress compact certificates.
-const CompactCertFilename = "compactcert.sqlite"
+// StateProofFileName is the name of the state proof database file.
+// It is used to track in-progress state proofs.
+const StateProofFileName = "stateproof.sqlite"
```

Comment:
Today I see a compactcert.sqlite file in my algod data directory — I assume they are empty but they are there. Do we need to change this? Maybe it is OK to maroon the old file forever?

Reply:
you are right. 'compactcert.sqlite' would be our there. However, they would be empty and won't be created on new nodes. maybe we can explicitly remove it in some script.